### DISCUSSION TITLE
HowTo Update

### DISCUSSION THREAD

**Discussion by W0lpertinger** [john30/ebusd#564] [Q&A]
Hi,
I didn't found a place where an update process was described.
System is running with ebusd-21.2_armhf-buster.deb package.
Goal should be to update without loosing config etc. if possible.

Thanks!

**Comment by john30** [comment]
depends on how you installed it in the first place. if you were using [ebusd-debian](https://github.com/john30/ebusd-debian/), then you can follow the instructions noted there.
otherwise you can use the initial installation method also to update the package
